draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -7885,24 +7885,29 @@ bytes=500-700,601-999
   <x:anchor-alias value="Accept-Ranges"/>
   <x:anchor-alias value="acceptable-ranges"/>
 <t>
-   The "Accept-Ranges" header field allows a server to indicate that it
-   supports range requests for the target resource.
+   The "Accept-Ranges" field in a response indicates whether an upstream
+   server supports range requests for the target resource.
 </t>
 <sourcecode type="abnf7230"><iref primary="true" item="Grammar" subitem="Accept-Ranges"/><iref primary="true" item="Grammar" subitem="acceptable-ranges"/>
   <x:ref>Accept-Ranges</x:ref>     = <x:ref>acceptable-ranges</x:ref>
-  <x:ref>acceptable-ranges</x:ref> = 1#<x:ref>range-unit</x:ref> / "none"
+  <x:ref>acceptable-ranges</x:ref> = 1#<x:ref>range-unit</x:ref>
 </sourcecode>
 <t>
-   An origin server that supports byte-range requests for a given target
-   resource &MAY; send
+   For example, a server that supports
+   <xref target="byte.ranges">byte-range requests</xref> can send the field
 </t>
 <sourcecode type="http-message">
 Accept-Ranges: bytes
 </sourcecode>
 <t>
-   to indicate what range units are supported. A client &MAY; generate range
-   requests without having received this header field for the resource
-   involved. Range units are defined in <xref target="range.units"/>.
+   to indicate that the request's target resource supports range requests,
+   thereby encouraging its use by the client for future partial requests.
+   Range units are defined in <xref target="range.units"/>.
```

Comment:
This isn't necessarily true; the target resource doesn't always support them if the header was added by a proxy.